python/lsst/ts/imsim/closed_loop_task.py

```diff
@@ -778,6 +778,28 @@ def _calc_wf_err_from_img(
 
         return list_of_wf_err
 
+    def _get_butler_inst_name(self, cam_type) -> str:
+        """Translate cam_type into suffix used by butler
+        in command line instructions.
+
+        Parameters
+        ----------
+        cam_type : lsst.ts.wep.utils.CamType
+            Camera type.
+
+        Returns
+        -------
+        str
+            Suffix attached to "LSST" to specify instrument to butler.
+        """
+
+        if cam_type in [CamType.LsstCam, CamType.LsstFamCam]:
+            butler_inst_name = "Cam"
+        elif cam_type == CamType.ComCam:
+            butler_inst_name = "ComCam"
```

Comment:
You may need to `raise` an error if someone passes an unknown type as the default (or `else`) case.

Reply:
I'll add that in, thanks.